.bumpedrc

```diff
@@ -0,0 +1,36 @@
+files: [
+  'package.json'
+]
+
+plugins:
+
+  prerelease:
+
+    'Linting config files':
+      plugin: 'bumped-finepack'
+
+    'Linting projects files':
+      plugin: 'bumped-terminal'
+      command: 'npm run lint'
+
+    'Preparing build':
+      plugin: 'bumped-terminal'
+      command: 'npm run prepublish'
+
+  postrelease:
+
+    'Generating CHANGELOG file':
+      plugin: 'bumped-changelog'
+
+    'Committing the new version':
+      plugin: 'bumped-terminal'
+      command: 'git add . && git commit -m "$newVersion releases"'
+
+    'Publishing at Github':
+      plugin: 'bumped-terminal'
+      command: 'git tag $newVersion && git push --tags && git push origin master'
```

Comment:
Can you write the complete *Publishing at Github* command following your approach?